app/src/main/res/values/strings.xml

```diff
@@ -475,9 +475,11 @@
 
     <!-- Label of the action that will add the current website to the home screen (shown in the confirmation dialog). -->
     <string name="dialog_addtohomescreen_action_add">Add</string>
+    <string name="dialog_addtohomescreen_action_yes">YES</string>
```

Comment:
Please copy over the comment from the 476 to above this string as well. Our automation extracts the strings file with the comment above each string to provide localizers context for translating.

Reply:
Same case for `dialog_addtohomescreen_action_no`.